server/src/apub/community.rs

```diff
@@ -45,6 +46,15 @@ impl ToApub for Community {
 
     Ok(group.extend(actor_props).extend(self.get_public_key_ext()))
   }
+
+  fn to_tombstone(&self) -> Result<Tombstone, Error> {
+    create_tombstone(
+      self.deleted,
+      &self.actor_id,
+      self.updated,
+      GroupType.to_string(),
+    )
+  }
```

Comment:
From what I see you can have this method return the HttpResponse directly, no need to use `create_apub_tombstone_response()` later.